gobblin-yarn/build.gradle

```diff
@@ -63,6 +63,33 @@ task testJar(type: Jar, dependsOn: testClasses) {
   from sourceSets.test.output
 }
 
+// create a single Jar with all dependencies
+task fatJar(type: Jar) {
```

Comment:
There is a gradle plugin to create fat jars. You can have a look li-gobblin:li-gobblin-fat/build.gradle and li-gobblin:build.gradle